### DISCUSSION TITLE
SSH Terrapin Prefix Truncation Weakness (CVE-2023-48795)

### DISCUSSION THREAD

**Discussion by Rob-Lingle-KR** [PowerShell/Win32-OpenSSH#2232] [General]
This vulnerability is fixed in Open SSH 9.6, do you know when you'll have a 9.6 available in this repo?  Thanks

SSH Terrapin Prefix Truncation Weakness (CVE-2023-48795)

**Comment by tgauth** [comment]
please see https://github.com/PowerShell/Win32-OpenSSH/issues/2189#issuecomment-1921958688

**Reply by Rob-Lingle-KR** [reply]
Thanks, sorry I did not search the issues.  I appreciate the quick response and guidance :-).
